tests/unit/configuration_subsystem/conftest.py

```diff
@@ -42,7 +42,7 @@ def _generate_config(params=None, setting_file_name=None, initial=True) -> Gener
 
     if setting_file_name:
         settings_file_path = os.path.join(TEST_FIXTURE_DIR, setting_file_name)
-        with open(settings_file_path) as file:
+        with open(file=settings_file_path, encoding="utf-8") as file:
```

Comment:
Why was this necessary? It's the default. Also, it's best to follow what the `tmp_path` does and operate on the pathlib objects. They also have `pathlib.Path.open()` if that's really necessary.

Reply:
pylint W1514 refers to PEP 597

Reply:
@webknjaz ^

Reply:
> pylint W1514 refers to PEP 597

Oh, interesting, I didn't realize. This raises another question — why didn't pytest reveal the warning. Ah! It's probably because we don't test on Python 3.10 (and maybe pytest also isn't set up right...)

But this is resolved for now, not really related to the PR.